torchvision/prototype/datapoints/_dataset_wrapper.py

```diff
@@ -237,6 +248,9 @@ def wrapper(idx, sample):
     return wrapper
 
 
+WRAPPER_FACTORIES.register(datasets.CocoCaptions)(identity_wrapper_factory)
```

Comment:
We need this now since

https://github.com/pytorch/vision/blob/707457050620e1f70ab1b187dad81cc36a7f9180/torchvision/datasets/coco.py#L60

Since the instantiation now no longer fails, our test try to wrap a sample and fail, since the wrapper of `CocoDetection` is not compatible.

Reply:
So the wrapper won't show the proper error message now?

> No wrapper exist for dataset class {dataset_cls.__name__}. Please wrap the output

Reply:
Without this, putting a `dataset = CocoCaptions(...)` into the wrapper will result in the wrapper for `CocoDetection` to be found, since `CocoCaptions` subclasses that. So yes, there won't be an error at instantiation, since we found a wrapper for a superclass.

Reply:
OK, as agreed offline it's probably not worth worrying too much about this detail - let's keep the identity wrapper

Reply:
As discussed offline, the identity wrapper here will work fully with the transforms and so we are keeping it although it doesn't work with the dataloader or the models. Basically, users needed custom code before to make it work, and it will continue to work.